packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -477,6 +521,34 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
                       canComment={canComment}
                       shouldFocusEditor={shouldFocusCommentEditor}
                     />
+                    {featureFlags.gatingEnabled && gatedGroups.length > 0 && (
+                      <CWBanner
+                        title="This topic is gated"
+                        body="Only members within the following group(s) can interact with this topic:"
+                        type="info"
+                        footer={
+                          gatedGroups && (
+                            <div className="gating-tags">
+                              {gatedGroups.map((t) => (
+                                <CWTag
+                                  key={t.id}
+                                  label={t.name}
+                                  type="referendum"
+                                />
+                              ))}
+                            </div>
+                          )
+                        }
+                        buttons={[
+                          {
+                            label: 'See all groups',
+                            onClick: () => navigate('/members?tab=groups'),
+                          },
+                          { label: 'Learn more about gating' },
```

Comment:
i am not sure, but shouldn't we redirect somewhere if they press on "lear more about gating" button?

If no, plz feel free to resolve this comment. Thanks.

Reply:
Unsure if that screen is created and it's not indicated in figma. 

CC: @zakhap any input here?